content/hi/docs/reference/glossary/service-account.md

```diff
@@ -0,0 +1,17 @@
+---
+title: ServiceAccount
```

Comment:
This should be in Hindi

Reply:
You can use `aka:` to also provide the original name.